src/components/CCPDashboard/HospitalOverview.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import {
+  makeStyles,
+  Box,
+  Card,
+  Grid,
+  Typography,
+  Select,
+  MenuItem,
+} from '@material-ui/core';
+import ExpandMoreRoundedIcon from '@material-ui/icons/ExpandMoreRounded';
+import ExpandLessRoundedIcon from '@material-ui/icons/ExpandLessRounded';
+import { Patient } from '../../graphql/queries/patients';
+import { TriageCard } from './TriageCard';
+import { PatientInfoTable } from './PatientInfoTable';
+import { ScanPatientButton } from './ScanPatientButton';
+import { CCPDashboardTabOptions } from './CCPDashboardPage';
+
+interface HospitalOverviewProps {
+  // eslint-disable-next-line react/no-unused-prop-types
+  eventId: string;
+  // eslint-disable-next-line react/no-unused-prop-types
+  ccpId: string;
```

Comment:
same thing here?

Reply:
They're required because the **edit patient profile page** has the route `/events/[event_id]/ccps/[ccp_id]/new` according to [this spreadsheet](https://docs.google.com/spreadsheets/d/1JTQlflqYCW8p78ceiyYTEJxTe1UuTgLT7OhHtPeyyV8/edit#gid=1720400799). `eventId` and `ccpId` are needed to be passed to `PatientInfoTable` > `PatientDetailsDialog` for the "Edit" button in that component which should link to the **edit patient profile page**.